core/src/test/scala/higherkindness/mu/rpc/srcgen/AvroScalaGeneratorArbitrary.scala

```diff
@@ -57,15 +53,16 @@ trait AvroScalaGeneratorArbitrary {
          |
          |$imports
          |
-         |final case class HelloRequest(arg1: String, arg2: Option[String], arg3: Seq[String])
+         |final case class HelloRequest(arg1: _root_.java.lang.String, arg2: _root_.scala.Option[_root_.java.lang.String], arg3: _root_.scala.List[_root_.java.lang.String])
```

Comment:
If the generator's outputs have changed (`Seq` to `List` etc.) let's make sure to document that in the appropriate project(s), with the relevant PRs marked as Breaking change if necessary.

Reply:
Will do!

Reply:
(Well, either that or make sure we're not changing the output if this wasn't intentional. Not sure if moving from `Seq` to `List` would be necessary.)

Reply:
Hmm, in this case it's the mu [`TList`](https://github.com/higherkindness/skeuomorph/blob/master/src/main/scala/higherkindness/skeuomorph/mu/codegen.scala#L187) type that is generating the list, so it'll be in line with what we expect on the proto side. If we wanted to keep `Seq` we'd need another update to skeuomorph, and then the proto side would change the opposite direction.

That said I think we should just go with `List`

Reply:
Makes sense. 👍